.github/workflows/build.yaml

```diff
@@ -31,25 +31,41 @@ jobs:
             - name: Setup Gradle
               uses: gradle/actions/setup-gradle@v3
               with:
-                gradle-version: 6.9.2
+                gradle-version: 8.9
             
             - name: Build Integration Tests Gradle source code
               run: |
-                gradle -b galasa-inttests-parent/build.gradle publish publishToMavenLocal \
+                set -o pipefail
+                gradle -b galasa-inttests-parent/build.gradle publish publishToMavenLocal --info \
                 -PsourceMaven=https://development.galasa.dev/gh/maven-repo/obr \
                 -PcentralMaven=https://repo.maven.apache.org/maven2/ \
-                -PtargetMaven=${{github.workspace}}/repo
+                -PtargetMaven=${{github.workspace}}/repo 2>&1 | tee gradle-build.log
+              
+            - name: Upload Maven Build Log
+              if: always()
+              uses: actions/upload-artifact@v4
+              with:
+                name: gradle-build-log
+                path: gradle-build.log

```

Comment:
I think this is meant to say Gradle here. Also can we rename the name of the artefact to "inttests-gradle-build-log" here so we can differentiate from what is being built below...

Reply:
done